core/templates/pages/community-dashboard-page/contributions-and-review/contributions-and-review.directive.ts

```diff
@@ -262,6 +265,10 @@ angular.module('oppia').directive('contributionsAndReview', [
           var _showTranslationSuggestionModal = function(
               targetId, suggestionId, contentHtml, translationHtml,
               reviewable) {
+            // We need to set the context here so that the rte fetches images
+            // for the give ENTITY_TYPE and entityId.
```

Comment:
nit: give --> given

Also do you mean targetId?

Reply:
The param of the function below is entityID so I've used entityID.

Reply:
Done!